mysql-test/t/percona_bug_ps260.test

```diff
@@ -0,0 +1,17 @@
+--source include/have_innodb.inc
+
+--echo #
+--echo # Loading the MySQL World sample database
+--echo # PS-260: An optimizer bug caused a crash when loading this SQL
+--echo #
+
+--disable_query_log
+--disable_result_log
+--echo # Loading data
+drop database if exists world;
+create database world;
+--exec $MYSQL  world 2>&1 < "./std_data/percona_bug_ps260_world200.sql"
```

Comment:
Inline the .sql file here instead of loading it

Reply:
Updated.